lib/facilities/ppms/client.rb

```diff
@@ -70,6 +82,21 @@ def specialties
 
       private
 
+      def trim_response_attributes(response)
```

Comment:
Would it make sense to add an unit test for `trim_response_attributes` ?

Reply:
since its a private method I don't want add direct tests for it. I am checking how this effects the other public methods. checkout `spec/lib/facilities/ppm/client_spec.rb`